src/itest/kotlin/nl/lifely/zac/itest/EnkelvoudigInformatieObjectRestServiceTest.kt

```diff
@@ -235,6 +245,130 @@ class EnkelvoudigInformatieObjectRestServiceTest : BehaviorSpec({
+                    )
+                    shouldContainJsonKey("informatieobjectTypeUUID")
+                    shouldContainJsonKey("identificatie")
+                    shouldContainJsonKeyValue("bestandsnaam", TEST_TXT_FILE_NAME)
+                    shouldContainJsonKeyValue("bestandsomvang", file.length().toString())
+                    shouldContainJsonKeyValue("formaat", TEXT_MIME_TYPE)
+                }
+                enkelvoudigInformatieObject2UUID = JSONObject(responseBody).getString("uuid")
+            }
+        }
+        When("the convert endpoint is called") {
+            val response = itestHttpClient.performPostRequest(
+                url = "$ZAC_API_URI/informatieobjecten/informatieobject/$enkelvoudigInformatieObject2UUID/" +
+                    "convert?zaak=$zaakProductaanvraag1Uuid",
+                requestBody = "".toRequestBody()
+            )
+            Then(
+                """
+                    the response should be OK and the informatieobject should be converted from TXT to PDF
+                    """
+            ) {
+                val responseBody = response.body!!.string()
+                logger.info { "Response: $responseBody" }
+                response.code shouldBe HTTP_STATUS_OK
+            }
+        }
+        When("the get enkelvoudiginformatieobject endpoint is called") {
+            val response = itestHttpClient.performGetRequest(
+                url = "$ZAC_API_URI/informatieobjecten/informatieobject/$enkelvoudigInformatieObject2UUID/"
+            )
+            Then(
+                """
+                    the response should be OK and should contain information about the document converted to PDF
+                    """
+            ) {
+                val responseBody = response.body!!.string()
+                logger.info { "Response: $responseBody" }
+                response.code shouldBe HTTP_STATUS_OK
+                with(responseBody) {
+                    shouldContainJsonKeyValue("auteur", TEST_USER_1_NAME)
+                    shouldContainJsonKeyValue("status", DOCUMENT_STATUS_DEFINITIEF)
+                    shouldContainJsonKeyValue("taal", "Engels")
+                    shouldContainJsonKeyValue("titel", DOCUMENT_FILE_TITLE)
+                    shouldContainJsonKeyValue(
+                        "vertrouwelijkheidaanduiding",
+                        DOCUMENT_VERTROUWELIJKHEIDS_AANDUIDING_OPENBAAR
+                    )
+                    shouldContainJsonKeyValue(
+                        "informatieobjectTypeOmschrijving",
+                        INFORMATIE_OBJECT_TYPE_BIJLAGE_OMSCHRIJVING
+                    )
+                    shouldContainJsonKey("informatieobjectTypeUUID")
+                    shouldContainJsonKey("identificatie")
+                    shouldContainJsonKeyValue("bestandsnaam", TEST_TXT_CONVERTED_TO_PDF_FILE_NAME)
+                    shouldContainJsonKeyValue("formaat", PDF_MIME_TYPE)
+                }
```

Comment:
Would this be better with `shouldEqualJson` ?

Reply:
yep. I just copied it from another block but you're right I guess. although we probably skip a few vars here..

Reply:
We can probably use the shortcuts we already have to ignore missing key-value pairs

Reply:
done :-)